yb-voyager/src/queryparser/query_parser.go

```diff
@@ -52,8 +71,36 @@ func IsPLPGSQLObject(parseTree *pg_query.ParseResult) bool {
 	return isPlPgSQLObject
 }
 
+func IsViewObject(parseTree *pg_query.ParseResult) bool {
+	_, isViewStmt := parseTree.Stmts[0].Stmt.Node.(*pg_query.Node_ViewStmt)
+	return isViewStmt
+}
+
+func IsMviewObject(parseTree *pg_query.ParseResult) bool {
+	createAsNode, isCreateAsStmt := parseTree.Stmts[0].Stmt.Node.(*pg_query.Node_CreateTableAsStmt)
+	return isCreateAsStmt && createAsNode.CreateTableAsStmt.Objtype == pg_query.ObjectType_OBJECT_MATVIEW
+}
+
+func GetSelectStmtQueryFromViewOrMView(parseTree *pg_query.ParseResult) (string, error) {
+	viewNode, isViewStmt := parseTree.Stmts[0].Stmt.Node.(*pg_query.Node_ViewStmt)
+	createAsNode, _ := parseTree.Stmts[0].Stmt.Node.(*pg_query.Node_CreateTableAsStmt)
```

Comment:
is createTableAsStmt the case for mview? Pls add a comment

Reply:
yes